<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>log_widget</class>
 <widget class="QWidget" name="log_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>721</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTextEdit" name="log_textEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="log_execute_pushButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="log_clear_pushButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="log_report_pushButton">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>3</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="script_editor_layout"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>log_widget</sender>
   <signal>destroyed()</signal>
   <receiver>log_widget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
